doc/source/batch-inference/doc_code/torch_image_batch_pretrained.py

```diff
@@ -0,0 +1,51 @@
+from typing import Dict, List
+import numpy as np
+
+import torch
+from torchvision import transforms
+from torchvision.models import resnet18
+
+import ray
+from ray.train.torch import TorchCheckpoint, TorchPredictor
+from ray.train.batch_predictor import BatchPredictor
+from ray.data.preprocessors import TorchVisionPreprocessor
+
+
+data_url = "s3://anonymous@air-example-data-2/1G-image-data-synthetic-raw"
+print(f"Running GPU batch prediction with 1GB data from {data_url}")
+dataset = ray.data.read_images(data_url, size=(256, 256)).limit(10)
+
```

Comment:
Why do you do size=(256,256) here? Please explain where it comes from.

Reply:
removed